<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1301</width>
    <height>856</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">/* === ОБЩЕЕ === */
QMainWindow {
    background-color: #0e1117;
    color: #e6edf3;
    font-family: Segoe UI, Inter, Arial;
    font-size: 12px;
}

/* === ГРУППЫ (блоки) === */
QGroupBox {
    background-color: #161b22;
    border: 1px solid #30363d;
    border-radius: 8px;
    margin-top: 8px;
}

QGroupBox::title {
    subcontrol-origin: margin;
    left: 12px;
    padding: 0 6px;
    color: #58a6ff;
    font-weight: bold;
}

/* === ТЕКСТОВЫЕ ПОЛЯ === */
QTextEdit,
QPlainTextEdit,
QLineEdit,
QLabel {
    background-color: #0d1117;
    border: 1px solid #30363d;
    border-radius: 6px;
    padding: 6px;
    color: #e6edf3;
    selection-background-color: #238636;
	font-size: 11pt;
}

/* === КНОПКИ === */
QPushButton {
    background-color: #238636;
    border: none;
    border-radius: 6px;
    padding: 6px 12px;
    color: white;
    font-weight: bold;
}

QPushButton:hover {
    background-color: #2ea043;
	font-size: 11pt;
}

QPushButton:pressed {
    background-color: #196c2e;
}

/* === ЛЕЙБЛЫ === */
QLabel {
    color: #e6edf3;
}

/* === SCROLLBAR (чтобы не резало глаза) === */
QScrollBar:vertical {
    background: #0d1117;
    width: 10px;
}

QScrollBar::handle:vertical {
    background: #30363d;
    border-radius: 5px;
}
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget#centralwidget {
    background-color: #0e1117;
}
</string>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="1" colspan="2">
     <widget class="QGroupBox" name="map">
      <property name="title">
       <string>MAP</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="0" column="0">
        <widget class="QLabel" name="map_pic">
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap>map.jpg</pixmap>
         </property>
         <property name="scaledContents">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
      <widget class="QPushButton" name="HELP">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>470</y>
         <width>71</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>HELP</string>
       </property>
      </widget>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QGroupBox" name="AI_helper">
      <property name="title">
       <string>AI</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <layout class="QVBoxLayout" name="AI_Layout">
         <item>
          <widget class="QTextEdit" name="Answ">
           <property name="readOnly">
            <bool>true</bool>
           </property>
           <property name="html">
            <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:11pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="quest">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="sendAI">
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>ico_send.png</normaloff>ico_send.png</iconset>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QGroupBox" name="Telemetry">
      <property name="title">
       <string>telemetry</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <widget class="QScrollArea" name="scrollArea">
         <property name="styleSheet">
          <string notr="true">background-color: #161b22;</string>
         </property>
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="Telematry_Content">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>459</width>
            <height>767</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">QWidget#centralwidget {
    background-color: #0e1117;
}
</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <layout class="QHBoxLayout" name="Name">
             <item>
              <widget class="QLabel" name="name_param">
               <property name="text">
                <string>Name of sattalite</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="name_value">
               <property name="text">
                <string>potuzhnolet</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="Number">
             <item>
              <widget class="QLabel" name="num_param">
               <property name="text">
                <string>serial №</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="num_value">
               <property name="text">
                <string>2718281828459045</string>
               </property>
               <property name="readOnly">
                <bool>true</bool>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="International_designator">
             <item>
              <widget class="QLabel" name="deignator_param">
               <property name="text">
                <string>international designator</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="designator_value">
               <property name="text">
                <string>84123 A</string>
               </property>
               <property name="readOnly">
                <bool>true</bool>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="inclination">
             <item>
              <widget class="QLabel" name="iclination_param">
               <property name="text">
                <string>inclination</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="inclination_value">
               <property name="text">
                <string>98.5105</string>
               </property>
               <property name="readOnly">
                <bool>true</bool>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="Right_ascenadion">
             <item>
              <widget class="QLabel" name="ascendation_param">
               <property name="text">
                <string>Right ascension of the ascending node</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="ascendation_value">
               <property name="text">
                <string>69.3305</string>
               </property>
               <property name="readOnly">
                <bool>true</bool>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="date_time_fraction">
             <item>
              <widget class="QLabel" name="date_time_pararm">
               <property name="text">
                <string>epoch year &amp; Julian day fraction</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="date_time_value">
               <property name="text">
                <string>86 50.28438588</string>
               </property>
               <property name="readOnly">
                <bool>true</bool>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="eccentricity">
             <item>
              <widget class="QLabel" name="eccentricity_param">
               <property name="text">
                <string>eccentricity</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="eccentricity_value">
               <property name="text">
                <string>001278</string>
               </property>
               <property name="readOnly">
                <bool>true</bool>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="argument_of_perrige">
             <item>
              <widget class="QLabel" name="perrige_param">
               <property name="text">
                <string>argument of perigee</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="perrige_value">
               <property name="text">
                <string>63.2828</string>
               </property>
               <property name="readOnly">
                <bool>true</bool>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="ballistic_cfcnt">
             <item>
              <widget class="QLabel" name="ballistic_param">
               <property name="text">
                <string>ballistic coefficient</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="ballistic_value">
               <property name="text">
                <string>0.00000140</string>
               </property>
               <property name="readOnly">
                <bool>true</bool>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="_2nd_dirrative">
             <item>
              <widget class="QLabel" name="dirrative_param">
               <property name="text">
                <string>2nd derivative of mean motion</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="dirrative_value">
               <property name="text">
                <string>0.0000</string>
               </property>
               <property name="readOnly">
                <bool>true</bool>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="mean_anomaly">
             <item>
              <widget class="QLabel" name="anomaly_param">
               <property name="text">
                <string>mean anomaly</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="anomaly_value">
               <property name="text">
                <string>296.9658</string>
               </property>
               <property name="readOnly">
                <bool>true</bool>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="Radiation_pressure">
             <item>
              <widget class="QLabel" name="pressure_param">
               <property name="text">
                <string>Raniation pressure coefficient</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="pressure_value">
               <property name="text">
                <string>67960-4</string>
               </property>
               <property name="readOnly">
                <bool>true</bool>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="mean_motion">
             <item>
              <widget class="QLabel" name="mean_motion_param">
               <property name="text">
                <string>mean motion</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="mean_motion_value">
               <property name="text">
                <string>14.24899292</string>
               </property>
               <property name="readOnly">
                <bool>true</bool>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="ephemeris_type">
             <item>
              <widget class="QLabel" name="ephemeris_param">
               <property name="text">
                <string>ephemeris type</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="ephemeris_value">
               <property name="text">
                <string>0</string>
               </property>
               <property name="readOnly">
                <bool>true</bool>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="element_number">
             <item>
              <widget class="QLabel" name="element_param">
               <property name="text">
                <string>element № &amp; Check sum</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="element_value">
               <property name="text">
                <string>5293</string>
               </property>
               <property name="readOnly">
                <bool>true</bool>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="rottation_speed">
             <item>
              <widget class="QLabel" name="rottation_param">
               <property name="text">
                <string>Revolution number at epoch &amp; Check sum </string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="rottation_value">
               <property name="text">
                <string>346978</string>
               </property>
               <property name="readOnly">
                <bool>true</bool>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="0" rowspan="2">
     <widget class="QGroupBox" name="console">
      <property name="title">
       <string>console</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_5">
       <item row="6" column="0">
        <widget class="QPushButton" name="sendPackButton">
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="text">
          <string>send pack</string>
         </property>
         <property name="flat">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QTextEdit" name="consoleTextEdit">
         <property name="font">
          <font>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
         <property name="html">
          <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:11pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;welcome&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLineEdit" name="commandLineEdit">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="sendCommandButton">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>ico_send.png</normaloff>ico_send.png</iconset>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
